extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -66,6 +79,35 @@ void build(CombinedIndexBuildItem index,
                 interfacesExtendingCrudRepository, indexIndex);
     }
 
+    private void detectAndLogSpecificSpringPropertiesIfExist() {
+        Config config = ConfigProvider.getConfig();
+        Map<String, String> springJpaToQuarkusOrmPropertiesMap = new HashMap<>();
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.show-sql", "quarkus.hibernate-orm.log.sql");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect ", "quarkus.hibernate-orm.dialect");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect.storage_engine",
+                "quarkus.hibernate-orm.dialect.storage-engine");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.generate-ddl", "quarkus.hibernate-orm.database.generation");
+
+        Iterable<String> iterablePropertyNames = config.getPropertyNames();
+        List<String> propertyNames = new ArrayList<String>();
+        iterablePropertyNames.forEach(propertyNames::add);
+        Pattern pattern = Pattern.compile("spring\\.jpa\\..*");
+        Matcher matcher = pattern.matcher("");
```

Comment:
I don't understand why you use this weird thing? Why not use `pattern.matcher(...).matches()` below?

Reply:
IIRC the `Matcher matcher = pattern.matcher("");` statement initialize a matcher and I use it in the loop: `propertyNames.stream().filter(s -> matcher.reset(s).matches()).collect(toList())` to avoid to have a new Matcher instance every time. The `reset` method resets the matching state internally in the Matcher and makes the Matcher search through the CharSequence passed as parameter. Make sense @gsmet ?

Reply:
I'm just totally unsure this is worth it.

Compiling the regexp would be far more beneficial IMHO. I have never seen this `reset()` pattern and I'm not sure adding to the complexity is worth it in this case as I'm unsure there is a big win.

Reply:
But I won't fight for it if you think it's really better. The other issues concern me more.

Reply:
I don't remember anymore why I did it like this so ok, let's do it as you propose.